packages/core/src/types/arazzo.ts

```diff
@@ -0,0 +1,345 @@
+    noneSourceDescriptionSchema,
+    arazzoSourceDescriptionSchema,
+  ],
+} as const;
+const sourceDescriptionsSchema = {
+  type: 'array',
+  items: sourceDescriptionSchema,
+} as const;
+const extendedOperation = {
+  type: 'object',
+  properties: {
+    path: { type: 'string' },
+    method: operationMethod,
+    sourceDescriptionName: { type: 'string' },
+    serverUrl: { type: 'string' },
+  },
+  additionalProperties: false,
+  required: ['path', 'method'],
+} as const;
+const parameter = {
+  type: 'object',
+  properties: {
+    in: { type: 'string', enum: ['header', 'query', 'path', 'cookie'] },
+    name: { type: 'string' },
+    value: {
+      oneOf: [{ type: 'string' }, { type: 'number' }, { type: 'boolean' }],
+    },
+    $ref: { type: 'string' },
+    style: { type: 'string' },
+    target: { type: 'string' },
+    required: { type: 'boolean' },
+    schema: {
+      type: 'object',
+      additionalProperties: true,
+    },
+    example: {
+      type: 'object',
+      additionalProperties: true,
+    },
+    examples: {
+      oneOf: [{ type: 'object' }, { type: 'object', properties: { $ref: { type: 'string' } } }],
+    },
+  },
+  additionalProperties: false,
+  required: ['in', 'name', 'value'],
+} as const;
+const parameters = {
+  type: 'array',
+  items: parameter,
+} as const;
+const infoObject = {
+  type: 'object',
+  properties: {
+    title: { type: 'string' },
+    description: { type: 'string' },
+    summary: { type: 'string' },
+    version: { type: 'string' },
+  },
+  additionalProperties: false,
+  required: ['title', 'version'],
+} as const;
+const replacement = {
+  type: 'object',
+  properties: {
+    target: { type: 'string' },
+    value: {
+      oneOf: [
+        { type: 'string' },
+        { type: 'object' },
+        { type: 'array' },
+        { type: 'number' },
+        { type: 'boolean' },
+      ],
+    },
+  },
+} as const;
+const requestBody = {
+  type: 'object',
+  properties: {
+    contentType: { type: 'string' },
+    payload: {
+      oneOf: [
+        { type: 'string' },
+        { type: 'object', additionalProperties: true },
+        { type: 'array' },
+        { type: 'number' },
+        { type: 'boolean' },
+      ],
+    },
+    encoding: { type: 'string' },
+    replacements: {
+      type: 'array',
+      items: replacement,
+    },
+  },
+  additionalProperties: false,
+  required: ['payload'],
+} as const;
+const criteriaObject = {
+  type: 'object',
+  properties: {
+    condition: { type: 'string' },
+    context: { type: 'string' },
+    type: { type: 'string', enum: ['regex', 'JSONPath'] },
+  },
+  required: ['condition'],
+  additionalProperties: false,
+} as const;
+const criteriaObjects = {
+  type: 'array',
+  items: criteriaObject,
+} as const;
+const inherit = {
+  type: 'string',
+  enum: ['auto', 'none'],
+} as const;
+const onSuccessObject = {
+  type: 'object',
+  properties: {
+    name: { type: 'string' },
+    type: { type: 'string', enum: ['goto'] },
```

Comment:
I think `end` is also supported here, but maybe you can double check? I'm reading it here https://github.com/OAI/Arazzo-Specification/blob/main/versions/1.0.0.md#success-action-object

Reply:
We don't support it in our implementation

Reply:
@RomanHotsiy / @tatomyr this actually brings another topic to discuss - as here we validate our Arazzo implementation, including some extensions and things we don't support.
Should we lint with pure Arazzo schema ? 
In this case we can keep two Arazzo schemas in cli and in Spot - as for sure they will be different.

Reply:
Could we support `end` when linting when we don't have support for it in Spot? If it's a lot of work then we can publish a list of limitations, but I think if we want to say that we support Arazzo, we should try to get as much of it as we can. (sorry, I realise this probably makes more work!).

Reply:
As discussed offline =>

-  I will update the schema to have pure Arazzo spec.
- Spot difference in spec should be handled with additional rules in the future (out of this PR scope)

Reply:
Arazzo schama is updated.

JSONSchema have some place for improvements - but good enough to digest Basic Arazzo examples, they are passing as a valid one now.